pkgs/development/libraries/rapidfuzz-cpp/default.nix

```
@@ -7,24 +7,19 @@
 
 stdenv.mkDerivation rec {
   pname = "rapidfuzz-cpp";
-  version = "1.0.2";
+  version = "1.0.3";
 
   src = fetchFromGitHub {
     owner = "maxbachmann";
     repo = "rapidfuzz-cpp";
     rev = "v${version}";
-    hash = "sha256-Tf7nEMXiem21cvQHPnYnCvOOLg0KBBnNQDaYIcHcm2g=";
+    hash = "sha256-8SJU+ERFRGkbGBmGJa5Ypetc3LPeytg5pR4S29RkvR8=";
   };
 
   patches = lib.optionals doCheck [
```

```suggestion
  patches = [
```

doCheck is always true